stable/bookstack/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 0.25.2
```

Comment:
appVersion is the version of upstream bookstack. Please change 'version' instead.

Reply:
Yeah, Just noticed that.